### PR TITLE
Php8

### LINKED ISSUE #38: Any chance to bump PHP req to 8.* speedily?
As per topic, can we extend the PHP requirement to 8.*?

### PR DESCRIPTION
**Pull request by jnkowa-gfk**
closes #38

### CODE REVIEW

**Comment by jnkowa-gfk** [comment]
@jaapio 
having this liv available on php8 is required to bring phpdocumentor to a php8 support.
```
composer require --dev phpdocumentor/phpdocumentor:dev-master
Running composer update phpdocumentor/phpdocumentor
Loading composer repositories with package information
Updating dependencies
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - phpdocumentor/flyfinder 1.0.0 requires php ^7.2 -> your php version (8.0.2) does not satisfy that requirement.
    - phpdocumentor/phpdocumentor dev-master requires phpdocumentor/flyfinder ^1.0 -> satisfiable by phpdocumentor/flyfinder[1.0.0].
    - Root composer.json requires phpdocumentor/phpdocumentor dev-master -> satisfiable by phpdocumentor/phpdocumentor[dev-master].
```

File: composer.json [hunk 1]
```diff
@@ -18,8 +18,8 @@
     }
   },
   "require": {
-    "php":              "^7.2",
-    "league/flysystem": "^1.0"
+    "php":              "^7.2||^8.0",
+    "league/flysystem": "^1.0||^2.0"
```
Comment (jaapio) [inline composer.json]: This library cannot be compatible with `league/flysystem:^2.0` Since this is a plugin, which have been removed in flysystem 2.

**Review by jaapio — COMMENTED** [review]
Thanks for this PR, there is something going on with Github actions. Which I don't understand yet. 

I think there is an invalid change in the composer.json regarding league/flysystem.

File: composer.json (hunk 1 above)
Comment (jnkowa-gfk) [reply composer.json]: i see.
reverted the change.

**Comment by ancarda** [comment]
@jaapio Can you re-run the failing GitHub Action? It looks like it just failed to download the PHPUnit PHAR file.

**Comment by douglasmiller** [comment]
@jaapio, I've got an open PR that includes the changes in this PR as well as updates to get the GitHub Actions to properly run again. Would it be possible to have that reviewed? I'd really like to be able to run my test suite using PHP 8!

**Pull request merged**